### DISCUSSION TITLE
How to get 'cabin check'

### DISCUSSION THREAD

**Discussion by uzytkownik** [flybywiresim/aircraft#4607] [Help] [answered]
It's not something that prevents me from flying but I see that I'm missing cabin check. Based on YT videos communication with cabin tend to be company specific. How to do it in FBW?

**Comment by andrejetcat** [comment, marked as answer]
Okay, technically you have to press at the overhead panel under section CALL either FWD or AFT. So your CABIN on the ECAM becomes green. In real life, the Purser (chief flight attendant) report "cabin ready" to the cockpit after , the name says, the cabin is ready for takeoff. 
In our company, we have also old A320´s , were the "ready message" only pops up on a seperate device. So there isen´t any point onto the ECAM. On the newer ones, the "cabin ready" from the pursers input replace YOUR press onto the FWD or AFT Button. That input comes from the Cabin itself. 

hope I could help and thats what you want to know

**Comment by ExampleWasTaken** [comment]
Adding to what adrejetcat said, certain conditions have to be met in order for the cabin to be ready. So don't wonder if you press the button and nothing happens right away.
